api/tests/Behat/bootstrap/v2/ReportManagement/ReportManagementTrait.php

```diff
@@ -0,0 +1,357 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Behat\v2\ReportManagement;
+
+use App\Tests\Behat\BehatException;
+use DateTime;
+
+trait ReportManagementTrait
+{
+    private array $baseCombinedHighReportCheckboxValuesAndTranslations = [
+        'decisions' => 'Decisions',
+        'contacts' => 'Contacts',
+        'visitsCare' => 'Visits and care',
+        'lifestyle' => 'Health and lifestyle',
+        'bankAccounts' => 'Accounts',
+        'moneyTransfers' => 'Money transfers',
+        'moneyIn' => 'Money in',
+        'moneyOut' => 'Money out',
+        'assets' => 'Assets',
+        'debts' => 'Debts',
+        'gifts' => 'Gifts',
+        'balance' => 'Accounts balance check',
+        'actions' => 'Actions you plan to take',
+        'otherInfo' => 'Any other information',
+        'documents' => 'Supporting documents',
+    ];
+
+    private array $orgCombinedHighExtraCheckboxes = [
+        'profDeputyCosts' => 'Deputy costs',
+        'profDeputyCostsEstimate' => 'Deputy costs estimate',
+    ];
+
+    private array $layCombinedHighExtraCheckboxes = [
+        'deputyExpenses' => 'Deputy expenses',
+    ];
+
+    protected string $reportStatus = '';
+
+    /**
+     * @When I manage the deputies :reportStatus report
+     */
+    public function iManageTheDeputiesSubmittedReport(string $reportStatus)
+    {
+        $this->iAmOnAdminClientDetailsPage();
+        $this->reportStatus = $reportStatus;
+
+        $reportId = 'completed' === $reportStatus ? $this->interactingWithUserDetails->getCurrentReportId() : $this->interactingWithUserDetails->getPreviousReportId();
+
+        $xpathLocator = sprintf(
+            "//a[contains(@href,'/admin/report/%s/manage')]",
+            $reportId
+        );
+
+        $reportLink = $this->getSession()->getPage()->find('xpath', $xpathLocator);
+        $reportLink->click();
+    }
+
+    /**
+     * @When I change the report type to :reportType
+     */
+    public function iChangeReportTypeTo(string $reportType)
+    {
+        $this->iAmOnAdminManageReportPage();
+        $this->assertInteractingWithUserIsSet();
+
+        $isLay = is_null($this->interactingWithUserDetails->getOrganisationName);
+
+        switch ($reportType) {
```

Comment:
Bear in mind. You're doing prod and lay here but not PA. Pa report types are different. This comment might be pointless.. haven't got to how you're using this yet ;)